### PR TITLE
Add release notes for Deploy 13.2.0, 14.1.0, 14.1.1 and Deploy Contrib 4.4.0, 10.3.0, 12.2.0, 13.2.0 and 14.1.0

### PR DESCRIPTION
**Pull request by ronaldbarendse**
## Description

Added release notes for Deploy 13.2.0, 14.1.0, 14.1.1 and Deploy Contrib 4.4.0, 10.3.0, 12.2.0, 13.2.0 and 14.1.0.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

- Deploy 13.2.0, 14.1.0, 14.1.1.
- Deploy Contrib 4.4.0, 10.3.0, 12.2.0, 13.2.0 and 14.1.0.

## Deadline (if relevant)

These releases are already out, so this can be merged immediately.

### CODE REVIEW

File: 10/umbraco-deploy/release-notes.md [hunk 1]
```diff
@@ -487,6 +491,10 @@ This section contains the release notes for Umbraco Deploy 4 and 10 including al
 <details>
 <summary>Version 4</summary>
 
+#### [4.4.0](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-4.4.0) (August 15th 2024)
+
+* All items from 4.4.0-rc1
```
Comment (jonat123) [inline 10/umbraco-deploy/release-notes.md:496]:
```suggestion
* Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
```

File: 10/umbraco-deploy/release-notes.md [hunk 2]
```diff
@@ -487,6 +491,10 @@ This section contains the release notes for Umbraco Deploy 4 and 10 including al
...
+
 #### [4.4.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-4.4.0-rc1) (July 19th 2024)
 
 * Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
```
Comment (jonat123) [inline 10/umbraco-deploy/release-notes.md:500]:
```suggestion
```

File: 10/umbraco-deploy/release-notes.md [hunk 3]
```diff
@@ -450,6 +450,10 @@ This section contains the release notes for Umbraco Deploy 4 and 10 including al
 
 ## Deploy Contrib
 
+#### [10.3.0](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-10.3.0) (August 15th 2024)
+
+* All items from 10.3.0-rc1
```
Comment (jonat123) [inline 10/umbraco-deploy/release-notes.md:455]:
```suggestion
* Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
```

File: 10/umbraco-deploy/release-notes.md [hunk 4]
```diff
@@ -450,6 +450,10 @@ This section contains the release notes for Umbraco Deploy 4 and 10 including al
...
+
 #### [10.3.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-10.3.0-rc1) (July 19th 2024)
 
 * Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
```
Comment (jonat123) [inline 10/umbraco-deploy/release-notes.md:459]:
```suggestion
```

File: 12/umbraco-deploy/release-notes.md [hunk 5]
```diff
@@ -122,14 +122,18 @@ This section contains the release notes for Umbraco Deploy 12 including all chan
 
 ## Deploy Contrib
 
+#### [12.2.0](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-12.2.0) (August 15th 2024)
+
+* All items from 12.2.0-rc1
```
Comment (jonat123) [inline 12/umbraco-deploy/release-notes.md:127]:
```suggestion
* Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
* Add DocTypeGridEditor Data Type configuration connector [#65](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/65)
```

File: 12/umbraco-deploy/release-notes.md [hunk 6]
```diff
@@ -122,14 +122,18 @@ This section contains the release notes for Umbraco Deploy 12 including all chan
...
+
 #### [12.2.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-12.2.0-rc1) (July 19th 2024)
 
 * Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
 * Add DocTypeGridEditor Data Type configuration connector [#65](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/65)
```
Comment (jonat123) [inline 12/umbraco-deploy/release-notes.md:132]:
```suggestion
```

File: 13/umbraco-deploy/release-notes.md [hunk 7]
```diff
@@ -17,6 +17,14 @@ If you are upgrading to a new major version you can find the details about the b
 
 This section contains the release notes for Umbraco Deploy 13 including all changes for this version.
 
+#### [13.2.0](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.2.0) (August 15th 2024)
+
+* All items from 13.2.0-rc1
```
Comment (jonat123) [inline 13/umbraco-deploy/release-notes.md:22]:
```suggestion
* Add migrators to support legacy Grid layout to Block Grid migration
```

File: 13/umbraco-deploy/release-notes.md [hunk 8]
```diff
@@ -17,6 +17,14 @@ If you are upgrading to a new major version you can find the details about the b
...
+* Fixed `Could not create Udi node: {Id} and entity type {EntityType}` exception when exporting tree nodes without children
+* Fixed deploy signatures not getting cleared when members are deleted [#230]( https://github.com/umbraco/Umbraco.Deploy.Issues/issues/230)
+* Allow members to be deployed when selected as items in a multi-node tree picker [#231](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/231)
+* Apply `ExcludedEntityTypes` configuration to disk operations [#232](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/232)
+
 #### [13.2.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.2.0) (July 19th 2024)
 
 * Add migrators to support legacy Grid layout to Block Grid migration
```
Comment (jonat123) [inline 13/umbraco-deploy/release-notes.md:30]:
```suggestion
```

File: 13/umbraco-deploy/release-notes.md [hunk 9]
```diff
@@ -104,6 +112,10 @@ This section contains the release notes for Umbraco Deploy 13 including all chan
 
 ## Deploy Contrib
 
+#### [13.2.0](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-13.2.0) (August 15th 2024)
+
+* All items from 13.2.0-rc1
```
Comment (jonat123) [inline 13/umbraco-deploy/release-notes.md:117]:
```suggestion
* Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
```

File: 13/umbraco-deploy/release-notes.md [hunk 10]
```diff
@@ -104,6 +112,10 @@ This section contains the release notes for Umbraco Deploy 13 including all chan
...
+
 #### [13.2.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-13.2.0-rc1) (July 19th 2024)
 
 * Skip empty pre-values [#63](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/63)
```
Comment (jonat123) [inline 13/umbraco-deploy/release-notes.md:121]:
```suggestion
```

File: 14/umbraco-deploy/release-notes.md [hunk 11]
```diff
@@ -53,6 +69,10 @@ This section contains the release notes for Umbraco Deploy 13 including all chan
 
 ## Deploy Contrib
 
+#### [14.1.0](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-14.1.0) (August 15th 2024)
+
+* All items from 14.1.0-rc1
```
Comment (jonat123) [inline 14/umbraco-deploy/release-notes.md:74]:
```suggestion
* Add Matryoshka Group Separator artifact migrator [#67](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/67)
```

File: 14/umbraco-deploy/release-notes.md [hunk 12]
```diff
@@ -53,6 +69,10 @@ This section contains the release notes for Umbraco Deploy 13 including all chan
...
+
 #### [14.1.0-rc1](https://github.com/umbraco/Umbraco.Deploy.Contrib/releases/tag/release-14.1.0-rc1) (July 19th 2024)
 
 * Add Matryoshka Group Separator artifact migrator [#67](https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/67)
```
Comment (jonat123) [inline 14/umbraco-deploy/release-notes.md:78]:
```suggestion
```

**Review by jonat123 — COMMENTED** [review]
@ronaldbarendse I have added suggestions to your PR as described above.

Let me know if you think it makes sense or not 😊

**Comment by ronaldbarendse** [comment]
Hi @jonat123! The RC releases are available publicly and for some versions, we might have multiple pre-releases that fix specific issues, so having them listed separately (without repeating them) would be the best IMHO 😄

**Comment by jonat123** [comment]
@ronaldbarendse Fair point :)

I will merge it then.

**Review by jonat123 — APPROVED** [review]

**Pull request merged**